script/Arcueid/Day 8/08_00_ARC08_2.txt

```diff
@@ -1,42 +1,42 @@
 [sha:71feefee92f58b34411714be601054237695ffe6]{
 -- Page 10005, Offset 19809.
 -- 「食事の前に話があります。
-"Before you eat, we have something to talk about.
+"Before you eat, there is something we must discuss.
 }
 [sha:96e8e9640908c27f12e92abe33657a1102a64c09]{
 -- Page 10005, Offset 19810.
 -- どうぞ、ソファーに座ってください」
-Please take a seat on the sofa."
+Please, take a seat on the sofa."
 }
 [sha:91dbe576cdd9a3f0788746c7cd3b917ca2951744]{
 -- Page 53, Offset 19811.
 -- 有無を言わさぬ、抵抗不可能の<言|こと>の<葉|は>なのだった。
-They were overwhelming words that I could not argue against.
+They were stern words, ones that I couldn't disobey.
 }
 [sha:2a4fc5d4cb2e185d8aa0571aad8a0ff13ad975e8]{
 -- Page 53, Offset 19812.
 -- 自業自得、<愚兄|ぐけい>死すべし、なんて単語が頭の中でかけ回る。
-Words such as 'you reap what you sow' and 'a foolish brother ought to die' revolve inside my head.
+Phrases such as 'you reap what you sow' and 'a foolish brother ought to die' revolve inside my head.
 }
 [sha:4977dc8abaf199a654c3e21f05911b69c483deb3]{
 -- Page 53, Offset 19813.
 -- ……観念して、ここで戦うしかないようだ。
-...I guess I have no choice but to accept it and take a stand.
+...I guess I have no choice but make my stand and accept my fate.
 }
 [sha:913eb14c0df9e132b179224fad4c2fef52193f94]{
 -- Page 54, Offset 19814.
 -- 「……わかった、出来るかぎり手短に頼む」
-"...Fine, please make it as brief as possible."
+"...Alright, please keep it brief if you can."
```

Comment:
this comes off as way less plead-y than it does in the jp. Maybe something like:
>...Alright, I just beg you to make it as short as you can.

over adjustment?